README.md

```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
 The Common Voice corpus consists of voice samples that were donated through Mozilla's [Common Voice](https://voice.mozilla.org/) Initiative.
 
-We provide an importer (`bin/import_cv.py`) which automates downloading and preparing the Common Voice corpus as such:
+We provide an importer (`bin/import_cv2.py`) which automates downloading and preparing the Common Voice (v2.0) corpus as such:
 
 ```bash
-bin/import_cv.py path/to/target/directory
+bin/import_cv2.py path/to/target/directory
```

Comment:
This is not the correct way to call `import_cv2.py`. Looking at the [code](https://github.com/mozilla/DeepSpeech/blob/master/bin/import_cv2.py#L139) it requires tow arguments.

Reply:
12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8